proposals/implicit-lambda-parameters-with-modifiers.md

```diff
@@ -103,3 +109,11 @@ We would also update the semantic specification to say:
 + is converted to a compatible delegate type or expression tree type, that type 
 + provides the parameter types (§10.7).
 ```
+
+#### Answer
+
+Neither attributes nor default parameter values will be supported without a fully-typed lambda.
+
+## Design meetings
```

Comment:
Let's please not include these links. The champion issue collates all the design meetings, and it is much more difficult for me to keep links in the repo up to date than it is to keep the champion issues up to date.